data/migrations/20211201172050__delete-old-Submissions-table.js

```diff
@@ -0,0 +1,76 @@
+exports.up = function (knex) {
+  return knex.schema
+    .alterTable('Writing', (t) => {
+      t.dropForeign('SubmissionID');
+    })
+    .alterTable('Drawing', (t) => {
+      t.dropForeign('SubmissionID');
+    })
+    .alterTable('Flags', (t) => {
+      t.dropForeign('SubmissionID');
+    })
+    .alterTable('Members', (t) => {
+      t.dropForeign('SubmissionID');
+    })
+    .alterTable('Points', (t) => {
+      t.dropForeign('SubmissionID');
+    })
+    .alterTable('Faceoffs', (t) => {
+      t.dropForeign('SubmissionID1');
+    })
+    .alterTable('Faceoffs', (t) => {
+      t.dropForeign('SubmissionID2');
+    })
+    .dropTableIfExists('Submissions', () => {})
+    .renameTable('Submissions-New', 'Submissions');
+};
+
+exports.down = function () {
+  exports.down = function (knex) {
+    return knex.schema
+      .renameTable('Submissions', 'Submissions-New')
+      .createTable('Submissions', (t) => {
+        t.increments('ID');
+        t.integer('ChildID')
+          .notNullable()
+          .unsigned()
+          .references('Children.ID')
+          .onUpdate('CASCADE')
+          .onDelete('RESTRICT');
+        t.integer('StoryID').notNullable().unsigned();
+        t.boolean('HasRead').defaultTo(false);
+        t.boolean('HasWritten').defaultTo(false);
+        t.boolean('HasDrawn').defaultTo(false);
+        t.integer('Complexity');
+        t.boolean('LowConfidence');
+
+        t.enu('Status', null, {
+          enumName: 'status',
+          existingType: true,
+          useNative: true,
+        }).defaultsTo('CLEAR');
+        t.integer('CohortID').notNullable().unsigned().defaultsTo(1);
+        t.unique(['ChildID', 'StoryID', 'CohortID']);
+      })
+
+      .alterTable('Writing', (t) => {
+        t.foreign('SubmissionID').references('Submissions.ID');
+      })
+      .alterTable('Drawing', (t) => {
+        t.foreign('SubmissionID').references('Submissions.ID');
+      })
+      .alterTable('Flags', (t) => {
+        t.foreign('SubmissionID').references('Submissions.ID');
+      })
+      .alterTable('Members', (t) => {
+        t.foreign('SubmissionID').references('Submissions.ID');
+      })
+      .alterTable('Points', (t) => {
+        t.foreign('SubmissionID').references('Submissions.ID');
+      })
+      .alterTable('Faceoffs', (t) => {
+        t.foreign('SubmissionID1').references('Submissions.ID');
+        t.foreign('SubmissionID2').references('Submissions.ID');
+      });
+  };
+};
```

Comment:
Line 76 has to be removed

Reply:
I apologize about that. I admit I did not test the down migration. But I did this time, and found some interesting results.

I played around the migrations and everything seem to work fine. 
![image](https://user-images.githubusercontent.com/77471489/147324264-644cf59f-2d6d-4d5a-9d37-a2a673c54bea.png)

However, after I ran the seed, I cannot migrate down without this error showing: 

![image](https://user-images.githubusercontent.com/77471489/147324338-eb305eab-fc53-45e2-9a29-2134ab4642b6.png)

I think it is because the newly created Submissions table in the down migration has no data. At any rate, all the changes you requested has been consolidated. Thank you for reviewing this PR :)

Reply:
Yes, the down migration will not work with data already in it. The only way to make that work is to delete data when down migrating but can cause problems in production.